posts/2022-08-04-jakarta-core-profile-22009-beta.adoc

```diff
@@ -0,0 +1,270 @@
+
+One of the new features is the addition of the link:https://jakarta.ee/specifications/expression-language/5.0/apidocs/jakarta.el/jakarta/el/methodreference[MethodReference] which "Provides information about the method to which a method expression resolves." This new class allows developers to access any annotations present on a particular method using the `getAnnotations()` method. There are also a few deprecations and clarifications within the Expression Language 5.0 Specification.
+
+[source,xml]
+----
+   <featureManager>
+        <feature>expressionLanguage-5.0</feature>
+    </featureManager>
+----
+
+More information regarding Jakarta Expression Language 5.0 can be  found on the 
+link:https://jakarta.ee/specifications/expression-language/5.0/[jakarta.ee site].
+
+[#microprofile]
+== MicroProfile OpenAPI 3.1 
+
+OpenAPI is a standardised way of documenting REST APIs in a JSON or YAML format. MicroProfile OpenAPI helps you generate and serve OpenAPI documentation for your REST applications built using JAX-RS or Jakarta restfulWS. This can be useful for developers to test out the API during development, or for people using the API in production.
+
+To use it, add the `mpOpenAPI-3.1` feature to your server.xml, deploy your rest application and then visit `/openapi` to view the OpenAPI documentation, or visit `/openapi/ui` for a user interface that lets you browse through it. You can enhance the documentation using annotations to add things like textual descriptions to each method. The annotations are available as a maven dependency:
+
+[source, xml]
+----
+<dependency>
+    <groupId>org.eclipse.microprofile.openapi</groupId>
+    <artifactId>microprofile-openapi-api</artifactId>
+    <version>3.1-RC2</version>
+</dependency>
+----
+
+Version 3.1 is a minor release which includes:
+
+* Some Jakarta Bean Validation annotations will now be read and their restrictions will be added to object schemas
+* Most annotations now include an `extension` attribute
+* `@SecurityRequirementsSet` can now be used to document operations where authentication is optional or where more than one authentication mechanism is required
+
+More information about MicroProfile OpenAPI can be found in:
+
+* link:{url-prefix}/guides/microprofile-openapi.html[Open Liberty MicroProfile OpenAPI guide]
+* link:{url-prefix}/docs/latest/documentation-openapi.html[Open Liberty OpenAPI documentation]
+* link:https://download.eclipse.org/microprofile/microprofile-open-api-3.1-RC2/apidocs/[MicroProfile OpenAPI 3.1 Javadoc]
+* link:https://download.eclipse.org/microprofile/microprofile-open-api-3.1-RC2/microprofile-openapi-spec-3.1-RC2.html[MicroProfile OpenAPI 3.1 Specification]
+
+[#password]
+== Password Utilities 1.1 
+
+The new `passwordUtilities-1.1` feature has been created to no longer start the `federatedRegistry-1.0` feature or the Jakarta Connectors feature. The new feature is part of core edition, whereas the `passwordUtilities-1.0` feature is part of base edition.
```

Comment:
What do "core edition" and "base edition" refer to? Editions of what?

Reply:
I've removed it as it's not applicable to Open Liberty